### DISCUSSION TITLE
Regarding Appium Server

### DISCUSSION THREAD

**Discussion by sg12057** [appium/appium#20235] [Q&A]
Hi Team,

Good day to you..

Regarding appium server I have one query.

1. For Appium Server are there any open CVE's patched for this
2. and if in future we get any issues do we have any support team to contact?
please answer my question as soon as possible.

**Comment by KazuCocoa** [comment]
If the report is reasonable for us, we'll attempt to fix such issues to the latest versions. We don't apply patches for older versions.

This is community-based OSS under https://github.com/appium/appium?tab=readme-ov-file#license license. We don't have a specific support team. If you wanted to have such a team or person, we'd recommend you find a consultation, etc, for it. https://discuss.appium.io/ is a primary place to ask questions. If you get any issues or possible bugs, please report them by following the https://github.com/appium/appium/issues/new/choose template.

**Comment by sg12057** [comment]
Hi Team,

Good day to you…

Regarding Appium Server I have some queries, below are my queries.


  1.  Does appium server have any open CVE’S patched.
  2.  If we get any issues for appium server are there any patches to fix those issues or can we contact support team to fix those issues.
  3.  I need to download appium server source file for MAC for the current version, please provide the link to download the source file for MACINTOSH platform.

Please respond as soon as possible as it’s a priority one.

Regards,
Sindusha
